src/css/lobby.styl

```diff
@@ -86,18 +86,19 @@
         padding: 0 15px 15px
 
         .rooms
-            overflow-y: auto
+            overflow-y visible;
```

Comment:
Missing a colon here?

Reply:
and I don't think you need the semi-colon, cuz stylus will handle that for you.

Reply:
how is stylus not showing an error in the build o.O?